airbyte-integrations/connectors/source-mysql/src/main/resources/spec.json

```diff
@@ -31,6 +31,10 @@
         "description": "Password associated with the username.",
         "type": "string",
         "airbyte_secret": true
+      },
+      "jdbc_url_params": {
+        "description": "Additional Parameters to pass to the jdbc url string when connecting to the database",
+        "type": "string"
```

Comment:
it seems like there is a very specific format expected here. can you add an example that shows adding multiple params?